### DISCUSSION TITLE
Maximum token issue

### DISCUSSION THREAD

**Discussion by Aiman-astios** [arc53/DocsGPT#175] [Q&A]
Hi, I'm new to this OpenAI thing.  
for context I use the same rst file to generate the faiss and pkl file and also ask the same question.

> This is the previous version of the docsgpt. 
![image](https://user-images.githubusercontent.com/110952815/225869397-3bbbf978-0697-4f6c-9cc4-a2fab2976e7b.png)

> This is the latest version of the docsgpt after the changes to the langchain and faiss file (officially).
![image](https://user-images.githubusercontent.com/110952815/225869283-4c8d8f95-c71b-4b54-bb30-fd9ad1c8e38c.png)

why do it get this error, i dont think that the message output from this rst file is more than 8000 words.

**Comment by dartpain** [comment]
Definetly an issue on our side, can you provide The documentation you used please. I want to reproduce this error.
If its confidential you can email it to me at alex@arc53.com please

**Reply by Aiman-astios** [reply]
sure can will email you the RST file

**Reply by Aiman-astios** [reply]
i have emailed you the rst file, but just in case i have also attached the rst file here.
[random-interview-1.zip](https://github.com/arc53/DocsGPT/files/11013249/random-interview-1.zip)

**Reply by Thayalan21** [reply]
@dartpain Is the token issue resolved i because i'm still facing it

**Reply by Thayalan21** [reply]
![image](https://user-images.githubusercontent.com/104500045/228779186-c5d2fe9f-bf96-4027-bf92-2ab6fc5f20c5.png)

**Comment by dartpain** [comment]
The context is too large to proces, there are two options, reduce k in the app.py or reduce max_token_length in worker.py

**Comment by Aiman-astios** [comment]
Thank you for your email, i will try it, hopefully one day i can contribute
to your community
On Thu, 30 Mar 2023 at 6:49 PM, Alex ***@***.***> wrote:

> The context is too large to proces, there are two options, reduce k in the
> app.py or reduce max_token_length in worker.py
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/arc53/DocsGPT/discussions/175#discussioncomment-5476294>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2OQC32L52ILCEMJ62QVZTDW6VQMBANCNFSM6AAAAAAV6JXZ6E>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 





[PRIVATE AND CONFIDENTIAL]




This e-mail contains confidential 
information. If you are not the intended addressee of this e-mail, any use, 
disclosure or copying of this e-mail is prohibited. If you receive this 
e-mail in error, please delete all copies of this e-mail, and contact the 
sender, immediately. Such confidentiality is not waived by reason of 
mistaken delivery to you. Any content herein that does not relate to the 
official business of Astiostech Group Of Companies, is neither given nor 
endorsed by said companies. Astiostech Group does not represent, warrant 
and/or guarantee that the integrity of this communication has been 
maintained, nor that it is free of any error, virus, interception or 
interference
